app/editor/src/components/layout/DefaultLayout.tsx

```diff
@@ -45,6 +57,52 @@ export const DefaultLayout: React.FC<ILayoutProps> = ({ name, children, ...rest
     };
   }, [keycloak, setToken]);
 
+  const onWorkOrder = React.useCallback(
+    (workOrder: IWorkOrderModel) => {
+      if (workOrder.workType === WorkOrderTypeName.FileRequest) {
+        const toastId = toastIds.find((t) => t.workOrderId === workOrder.id);
+        if (toastId) {
+          toast.dismiss(toastId.toastId);
+          setToastIds((state) => state.filter((t) => t.toastId !== toastId?.toastId));
+        }
+
+        if (workOrder.status === WorkOrderStatusName.InProgress) {
+          const id = toast.loading(
+            `File copy request in progress. File: ${workOrder.configuration.path}`,
+            {
+              closeButton: true,
+            },
+          );
+          setToastIds((state) => [
+            ...state,
+            {
+              toastId: id,
+              workOrderId: workOrder.id,
+            },
+          ]);
+        } else if (workOrder.status === WorkOrderStatusName.Completed) {
+          toast.success(() => (
+            <div>
+              File copy request complete: File:{' '}
+              <Link
+                to={`/storage/locations/1?path=/_tmp/${workOrder.configuration.path}`} // TODO: shouldn't hardcode location.
```

Comment:
Provides a way to click the link and go directly to the file that has been copied to the API.